V3-Upgrade-Guide.md

```diff
@@ -36,58 +32,40 @@ Depending on if you're using `npm` or `yarn`, run `npm install` or `yarn` in the
 Running your tests or deploying your application might fail at this point in time if you need to adapt to any breaking changes.
 We recommend updating your applications in one commit or pull request and making sure everything still works using your existing test suite.
 
-### Important Dependency Updates
+### Important Dependency Updates <!-- omit from toc -->
 
 The [axios HTTP client](https://github.com/axios/axios) has been updated from version 0.27 to 1.2.
 
-### Support Node 18
-Node 18 is the current long term support (LTS) version. 
-Previous node versions will reach their end of life within the next year (see [node.js release schedule](https://github.com/nodejs/Release#release-schedule)). 
-Therefore, all SAP Cloud SDK for JavaScript libraries will switch to node 18 as the **minimum** supported node version.
-If you run an older (<18) node version, please update to a newer version.
-You can find a list of breaking changes in the news section of the node.js [website](https://nodejs.org/en/blog/).
-
-### Update EcmaScript Runtime
-
-We changed the compilation target of our source code from `es2019` to `es2021`.
-Depending on your configuration this may lead to compilation errors (TypeScript) or runtime errors in (JavaScript).
+## Update to Node 18 or Newer
 
-### Adjust Operation Names in OData Generated Clients
+All SAP Cloud SDK for JavaScript libraries now support node 18 (LTS) as the **minimum** node version.
+If you are using a node version older than 18, update your runtime environment to a newer version.
+Note, that the transpilation target of the SDK changed from `es2019` to `es2021`.
 
-Rules for naming of OData operations (actions or functions) in the generated client have been changed.
-This applies to bound and unbound operations.
-If an operation begins with an `underscore` symbol, the `_` will be removed from the resulting generated client code.
-To adjust the names, search in `function-import.ts` and `action-import.ts` files in your generated client code for any operation starting with `_`.
-Similarly, to adjust the names of bound operations of an entity, search in the respective entity's `.ts` file, e.g., `TestEntity.ts`.
+## Replace Removed Functionality
 
-### Check for removed deprecated functions and replace them if required
-
-While the SAP Cloud SDK maintains backwards compatibility within a major version where possible, a new major release breaks compatibility where required to simplify the programming interface.
-Most of the removed functions had been deprecated before, so ideally they are not used anymore.
+Most of the removed functionality had been deprecated in version 2, so ideally they are not used anymore.
```

Comment:
[pp] The `so ideally they are not used anymore` sounds strange to me. Perhaps be more bold and say something like  "the deprecated things have been removed."

Reply:
I also stumbled upon that. I just removed that part of the sentence. Is it better?